pkg/ddevapp/app_compose_template.yaml

```diff
@@ -306,8 +306,6 @@ services:
       - XHGUI_PDO_USER=db
       - XHGUI_PDO_PASS=db
       - TZ={{ .Timezone }}
-    links:
-      - db
```

Comment:
Deleted this for Podman, it didn't help, but still - `links:` are not required nowadays.